sdk/samples/sender_app.c

```diff
@@ -20,7 +20,7 @@
 #define DEFAULT_RECV_IP "127.0.0.1"
 #define DEFAULT_RECV_PORT "9001"
 #define DEFAULT_SEND_IP "127.0.0.1"
-#define DEFAULT_SEND_PORT "9001"
+#define DEFAULT_SEND_PORT "9002"
```

Comment:
Why do you change this value?

Reply:
see https://github.com/OpenVisualCloud/Media-Communications-Mesh/pull/178#discussion_r1760590068